config/config.go

```diff
@@ -123,6 +125,10 @@ type Config struct {
 	lock        sync.RWMutex
 	AcceptAddrs []string `json:"acceptAddrs"`
 	AdminAddrs  []string `json:"adminAddrs"`
+
+	// nconnect network
+	NodeName       string `json:"nodeName,omitempty" long:"node-name" description:"Node name, will be used as nConnect network node name"`
```

Comment:
(network member only) Node name that will be used as to join a network

Reply:
Yes, modified.